lib/Bio/EnsEMBL/DataCheck/Checks/MTCodonTable.pm

```diff
@@ -0,0 +1,66 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::MTCodonTable;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Utils qw/sql_count/;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+
+use constant {
+  NAME           => 'MTCodonTable',
+  DESCRIPTION    => 'MT seq region had associated seq_region attribute '
+    . 'and correct codon table',
+  GROUPS         => ['core_handover'],
+  DATACHECK_TYPE => 'critical',
+  DB_TYPES       => ['cdna', 'core', 'funcgen', 'otherfeatures', 'rnaseq', 'variation'],
+  TABLES         => ['seq_region', 'seq_region_attrib']
```

Comment:
There are tables that are in the queries which aren't listed here; need to be comprehensive, attrib_type is unlikely to change, but if it does we could miss out on re-running the datacheck when we need to.